.circleci/config.yml

```diff
@@ -51,7 +51,10 @@ jobs:
 
   build-macos:
     macos: # We're using a Mac host because that's the only way to build for Xamarin iOS
-      xcode: "12.5.1"
+      # If the xcode version (or possibly resource class) is updated, we'll need to install JDK 11 explicitly for use
+      # with Xamarin.
+      xcode: "13.4.1"
+    resource_class: macos.x86.medium.gen2
```

Comment:
Just FYI: This will be deprecated in June 2024
https://discuss.circleci.com/t/macos-intel-support-deprecation-in-january-2024/48718

I think you can skip specify this and the build will select whatever is available or explicitly set `macos.m1.medium.gen1` unless you need an x86 host.

Reply:
If I remember correctly, I ran into this resource class deprecation in the iOS SDK and did not have to specify the resource class.  https://github.com/launchdarkly/ios-client-sdk/pull/302/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47